rollup.config.js

```diff
@@ -29,13 +32,23 @@ const config = mode => ({
         if (source === 'ReactDOM') {
           return {id: 'react-dom', external: true};
         }
+        if (source === 'ReactNative') {
+          return {id: 'react-native', external: true};
+        }
         return null;
       },
     },
-    nodeResolve(),
+    nodeResolve({
+      extensions: target === 'native' ? ['.native.js', '.js'] : undefined,
```

Comment:
This will resolve `.native.js` files over `.js` files with the same name. This is how we have Rollup to import `ReactBatchedUpdates.native.js` when building for RN.